Library/Formula/tesseract.rb

```diff
@@ -65,6 +65,12 @@ class Tesseract < Formula
   end
 
   def install
+    if build.with? "training-tools"
+      icu4c = Formula["icu4c"]
+      ENV.append "CFLAGS", "-I#{icu4c.include}"
```

Comment:
`Formula["icu4c"].opt_include` will point to `HOMEBREW_PREFIX/opt/icu4c/include` instead of the Cellar path so it won't break on upgrades.